src/config/dynamic-theme-fixes.config

```diff
@@ -135,6 +135,18 @@ IGNORE INLINE STYLE
 
 ================================
 
+*.ubereats.com
+
+INVERT
+img[alt*='Home']
+body > #root > #wrapper > #main-content h1 + div
+body > #root > #wrapper > div[class^='c']
```

Comment:
I think if you remove `body > #root` here and just leave `#wrapper` as control you should be good to go.

Reply:
Quite possibly, yes!    
But I didn't want to repeatedly comb through the whole functionality of the site to try to whittle down the selector to the minimum required (particularly since my food was arriving!).    
All it'd take is for one part of one page under one circumstance to be similar enough inside it's #wrapper, and then the selector is interfering with parts that work just fine.    

If you're asking me why I didn't, I preferred to err on the most cautious side from the beginning, and I ran out of momentum when my food arrived. 
If you're asking me to do it and see what happens because the selectors for fixes should be as minimal and efficient as possible as a matter of standard practice/policy, I can do that no worries, but it'll have to wait until I order something from Uber again to test it properly.

Reply:
Got it! Just a minor issue nothing breaking 😄